test/jasmine/tests/hover_label_test.js

```diff
@@ -749,7 +749,7 @@ describe('hover on fill', function() {
         }).then(function() {
             return assertLabelsCorrect([237, 218], [266.75, 265], 'trace 1');
         }).then(function() {
-            return assertLabelsCorrect([237, 251], [247.7, 254], 'trace 0');
+            return assertLabelsCorrect([237, 240], [247.7, 254], 'trace 0');
```

Comment:
cc @alexcjohnson these cases :arrow_up: were tuned for FF on CI and Chrome locally. Now, they work locally in FF and Chrome as well as FF on CI.